backend/infrahub/core/branch/tasks.py

```diff
@@ -248,3 +251,49 @@ async def validate_branch(branch: str) -> State:
     if conflicts:
         return Failed(message="branch has some conflicts")
     return Completed(message="branch is valid")
+
+
+# Note 1: We could use a `data` BaseModel instead of a dict here (as it's done for other flows),
+# but it would require duplicating `BranchCreateInput` class definition. The initial issue is that
```

Comment:
Agreed, let's create a new data structure even if it's a dup of `BranchCreateInput` it's best than to send a dict without information